osf/models/mixins.py

```diff
@@ -2013,14 +2013,6 @@ def confirm_ham(self, save=False):
             self.save()
 
     def _get_spam_content(self, saved_fields):
-        """
-        This function retrieves retrieves strings of potential spam from various DB fields. Also here we can follow
```

Comment:
Why was this docstring removed?

Reply:
I meant to remove the docstring on check_spam